docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
```

Comment:
Not a part of this PR, Line 26, full stop after ending of problem statement, and capital T for the problem. 
the problem is likely to be `virtlogd` service not running, you can check it with>  The problem is likely to be that the `virtlogd` service is not running, you can check it with....